include/tvm/runtime/ndarray.h

```diff
@@ -33,6 +33,8 @@
 namespace tvm {
 namespace runtime {
 
+size_t GetDataAlignment(const DLTensor& arr);
```

Comment:
Previously this API was private so it was written in a not very thoughtful way. I am debating whether or not we should include it as a public API. Perhaps just have two inlined version of this function for now internally. As the behavior was mainly return kTempAllocaAlign. If it is a public API, we need to properly document it.

Reply:
Yeah, I can make them local functions.